boards/st/b_u585i_iot02a/b_u585i_iot02a_stm32u585xx_ext_flash_app.yaml

```
@@ -0,0 +1,10 @@
+identifier: b_u585i_iot02a/stm32u585xx/xip
```

```suggestion
b_u585i_iot02a/stm32u585xx/ext_flash_app
```
(issue reported by CI build tests)

---

done